lib/extras/codec_apng.cc

```diff
@@ -347,7 +350,7 @@ int processing_data(png_structp png_ptr, png_infop info_ptr, unsigned char* p,
 
   if (setjmp(png_jmpbuf(png_ptr))) {
     png_destroy_read_struct(&png_ptr, &info_ptr, 0);
-    return 1;
+    recounturn 1;
```

Comment:
Are you sure it's not a typo? What is it supposed to do?

Reply:
it was a typo :)